### ISSUE TITLE
I can not  get a good result through your code, loss doesnot go down

### ISSUE THREAD

**Issue by sjq19960802** [rharish101/PongRL#2]
Loss doesnot go down
![image](https://user-images.githubusercontent.com/32811256/73901741-a8c27300-48ce-11ea-8ee4-5589957d5acb.png)

**Comment by rharish101** [comment]
Did you verify from the videos that the agent is performing poorly? Sometimes the agent gets stuck in poor local minima, and training refuses to go forward. I'm still trying to get the training to stabilize; for now, try experimenting with the hyper-params for stability. If you know of any concrete way to stabilize the training, please file a pull request.

**Comment by sjq19960802** [comment]
Yes, i run the test.py and get this video. It seems that the agent is still performing poorly.&nbsp;


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Harish Rajagopal"<notifications@github.com&gt;;
发送时间:&nbsp;2020年2月6日(星期四) 中午1:15
收件人:&nbsp;"rharish101/PongRL"<PongRL@noreply.github.com&gt;;
抄送:&nbsp;"631592160"<631592160@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [rharish101/PongRL] I can not  get a good result through your code, loss doesnot go down (#2)




Did you verify from the videos that the agent is performing poorly? Sometimes the agent gets stuck in poor local minima, and training refuses to go forward. I'm still trying to get the training to stabilize; for now, try experimenting with the hyper-params for stability. If you know of any concrete way to stabilize the training, please file a pull request.
